### PR TITLE
Preserve imunify license file during elevation

### PR DESCRIPTION
**Pull request by toddr**
Resolves GH #59

By submitting pull requests to this repo, I agree to the Contributor License Agreement which can be found at: https://github.com/cpanel/elevate/blob/main/docs/cPanel-CLA.pdf

### CODE REVIEW

File: elevate-cpanel [hunk 1]
```diff
@@ -911,12 +913,19 @@ sub capture_imunify_features {
 
     my $output   = Cpanel::SafeRun::Simple::saferunnoerror( IMUNIFY_AGENT, qw{features list} );
     my @features = map { s/\s+//g; $_ } grep { m/\S/ } split( "\n", $output );
+
+    if ( -f IMUNIFY_LICENSE_FILE ) {
+        File::Copy::move( IMUNIFY_LICENSE_FILE, '/var/cpanel' );
```
Comment (atoomic) [inline elevate-cpanel:918]: this can be confusion this is not using `IMUNIFY_LICENSE_BACKUP`

File: elevate-cpanel [hunk 2]
```diff
@@ -178,7 +178,9 @@ use constant IGNORE_OUTDATED_SERVICES_FILE => q[/etc/cpanel/local/ignore_outdate
 use constant LEAPP_REPORT_JSON => q[/var/log/leapp/leapp-report.json];
 use constant LEAPP_REPORT_TXT  => q[/var/log/leapp/leapp-report.txt];
 
-use constant IMUNIFY_AGENT => '/usr/bin/imunify360-agent';
+use constant IMUNIFY_AGENT          => '/usr/bin/imunify360-agent';
+use constant IMUNIFY_LICENSE_FILE   => '/var/imunify360/license.json';
+use constant IMUNIFY_LICENSE_BACKUP => '/var/cpanel/imunify-backup-license.json';
```
Comment (atoomic) [inline elevate-cpanel:183]: consider using `ELEVATE_BACKUP_DIR`

**Review by atoomic — CHANGES_REQUESTED** [review]
please use `ELEVATE_BACKUP_DIR` and fixup the backup location

File: elevate-cpanel (hunk 1 above)
Comment (toddr) [reply elevate-cpanel]: incomplete conversion. try now.

**Review by atoomic — APPROVED** [review]

**Pull request merged**